packages/ozone/src/daemon/context.ts

```diff
@@ -67,6 +70,17 @@ export class DaemonContext {
       appviewAgent,
       createAuthHeaders,
     )
+    const teamService = TeamService.creator()
+    const teamProfileSynchronizer = new TeamProfileSynchronizer(
+      backgroundQueue,
+      {
+        createAuthHeaders: (method: string) =>
```

Comment:
This reads a bit weird but basically can't come up with a better, more context aware name that would work elsewhere as well as here.

Reply:
I like that pretty well too 👍